mailbox/backup/src/main/java/org/apache/james/mailbox/backup/SerializedMailboxId.java

```diff
@@ -0,0 +1,55 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+package org.apache.james.mailbox.backup;
+
+import org.apache.james.mailbox.model.MailboxId;
+
+import com.google.common.base.Objects;
+
+public class SerializedMailboxId {
+    private final String value;
+
+    public SerializedMailboxId(String value) {
+        this.value = value;
+    }
+
+    public SerializedMailboxId(MailboxId mailboxId) {
+        this.value = mailboxId.serialize();
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (o instanceof SerializedMailboxId) {
+            SerializedMailboxId that = (SerializedMailboxId) o;
+            return Objects.equal(value, that.value);
+        } else {
```

Comment:
You can drop it

Reply:
Fixed in a previous  PR. Will be rebased